### PR TITLE
Colors hardening

### PR DESCRIPTION
**Pull request by gregaubert**
See:

* [SONAR-12543](https://jira.sonarsource.com/browse/SONAR-12543)
* [SONAR-12539](https://jira.sonarsource.com/browse/SONAR-12539)
* [SONAR-12541](https://jira.sonarsource.com/browse/SONAR-12541)
* [SONAR-12545](https://jira.sonarsource.com/browse/SONAR-12545)
* [SONAR-11601](https://jira.sonarsource.com/browse/SONAR-11601)

Related to https://github.com/SonarSource/sonarcloud-core/pull/201

**Checklist**

* [ ] ~Write/update ITs~
* [x] Functional validation
* [ ] ~Documentation update~
* [x] Update changelog

### CODE REVIEW

**Comment by gregaubert** [comment]
@jeremy-davis-sonarsource Can you have look ? It's related to a PR on SC for which I pinged you too to cherry-pick what's meaningful for you. Don't hesitate to tell me if I should ping someone else from SQ team instead.

File: components/controls/GlobalMessages.tsx [hunk 1]
```diff
@@ -60,14 +60,14 @@ export class GlobalMessage extends React.PureComponent<{
     const { message } = this.props;
     return (
       <div
-        className={classNames('process-spinner', 'shown', {
-          'process-spinner-failed': message.level === 'ERROR',
-          'process-spinner-success': message.level === 'SUCCESS'
+        className={classNames('process-message', 'shown', {
+          'process-message-failed': message.level === 'ERROR',
+          'process-message-success': message.level === 'SUCCESS'
```
Comment (jeremy-davis-sonarsource) [inline components/controls/GlobalMessages.tsx]: Why not make it a styled-component?

**Review by jeremy-davis-sonarsource — APPROVED** [review]
LGTM! I have a PR as well that we can combine in v `0.0.32`

File: components/controls/GlobalMessages.tsx (hunk 1 above)
Comment (gregaubert) [reply components/controls/GlobalMessages.tsx]: Ahhhh right! We can make styled-component! I had forgotten!
I'll have a look to transform it!

**Comment by gregaubert** [comment]
Yeah sure let's combine and do only one release :)

File: components/controls/GlobalMessages.tsx [hunk 2]
```diff
@@ -17,37 +17,85 @@
...
+const appearAnim = keyframes`
+  from {
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
+`;
+
+const Message = styled.div<{ level: Message['level'] }>`
```
Comment (jeremy-davis-sonarsource) [inline components/controls/GlobalMessages.tsx]: use `Pick<Message, 'level'>` ?

File: components/controls/GlobalMessages.tsx [hunk 3]
```diff
@@ -17,37 +17,85 @@
...
+  opacity: 0;
+  animation: ${appearAnim} 0.2s ease forwards;
+
+  & + & {
+    margin-top: calc(${({ theme }) => theme.sizes.gridSize} / 2);
+    border-radius: 3px;
+  }
+`;
+
+const CloseButton = styled(ClearButton)<{ level: Message['level'] }>`
```
Comment (jeremy-davis-sonarsource) [inline components/controls/GlobalMessages.tsx]: same as above

File: components/controls/GlobalMessages.tsx [hunk 4]
```diff
@@ -17,37 +17,85 @@
...
+  padding: 0 30px 0 10px;
+  line-height: ${({ theme }) => theme.sizes.controlHeight};
+  border-radius: 0 0 3px 3px;
+  box-sizing: border-box;
+  color: #ffffff;
+  background-color: ${({ level, theme }) =>
+    level === 'SUCCESS' ? theme.colors.green : theme.colors.red};
+  text-align: center;
+  opacity: 0;
+  animation: ${appearAnim} 0.2s ease forwards;
```
Comment (jeremy-davis-sonarsource) [inline components/controls/GlobalMessages.tsx:80]: Great solution to transition issue!

**Review by jeremy-davis-sonarsource — APPROVED** [review]
LGTM! Just minor typing nitpicking

File: components/controls/GlobalMessages.tsx (hunk 2 above)
Comment (gregaubert) [reply components/controls/GlobalMessages.tsx]: Ok, any particular reason ?

**Pull request merged**